packages/libaudit/Cargo.toml

```diff
@@ -12,8 +12,8 @@ path = "../packages.rs"
 releases-url = "https://github.com/linux-audit/audit-userspace/releases"
 
 [[package.metadata.build-package.external-files]]
-url = "https://github.com/linux-audit/audit-userspace/archive/v3.1.2/audit-userspace-3.1.2.tar.gz"
-sha512 = "d5b05686aedd6eaaced000778580bd3e96e38c020534038d6b1ee6de96dd7687c2f3a52a2ae75be8b93c173b286a56dbc92231906c68fda79b113dfcaad6da84"
+url = "https://github.com/linux-audit/audit-userspace/archive/v3.1.3/audit-userspace-3.1.3.tar.gz"
```

Comment:
It's fine to stay on 3.x for now, but it should also be possible to move up to audit 4 - or if it's not, we should at least have tried it and have a sense of what's broken.

Reply:
Yes.I have added libaudit and libglib to major version.